src/api-v1.js

```diff
@@ -414,6 +414,45 @@ api.declare({
 });
 
 api.declare({
+  method:     'delete',
+  route:      '/killWorker',
```

Comment:
this is not a REST endpoint.  This is also not the correct service to have this endpoint.

The correct URL to use is:

```
https://ec2-manager.taskcluster.net/v1/region/{WORKERGROUP}/instance/{WORKERID}
```